src/ontology/mondo-ingest.Makefile

```diff
@@ -28,6 +28,21 @@ dependencies:
 	$(MAKE) pip-sssom
 	$(MAKE) pip-semsql
 
+####################################
+### General ########################
```

Comment:
## `%.db` to new "General" section
The previous `*.db` goals were in the "lexmatch" section even though not all of them were lexmatch related. I kept the `merged.db` goal there since lexmatch is all that depends on it, but I moved `%.db` here to this new general section, as there didn't look like there was anywhere else appropriate to put it.